src/clientsettingsdlgbase.ui

```diff
@@ -455,7 +462,7 @@
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
-             <widget class="QLineEdit" name="edtNewClientLevel"/>
```

Comment:
Not sure, but is this deletion intended?

Reply:
I think it should be correct, it might just look strange in the diff:

- Technically, I add a new QLineEdit for edtInputGain.
- To a diff algorithm it looks like I'm changing the existing edtNewClientLevel (the line you marked) to edtInputGain (the line below) and add a new QLineEdit for the old, existing edtNewClientLevel (see line 477).

Reply:
Ok. Seems you're right!

I looked into the file and it's ok there.